src/env.h

```diff
@@ -154,6 +154,7 @@ constexpr size_t kFsStatsBufferLength = kFsStatsFieldsNumber * 2;
   V(change_string, "change")                                                   \
   V(channel_string, "channel")                                                 \
   V(chunks_sent_since_last_write_string, "chunksSentSinceLastWrite")           \
+  V(clone_unsupported_type_str, "Cannot transfer object of unsupported type.") \
```

Comment:
```suggestion
  V(clone_unsupported_type_str, "Cannot transfer object of unsupported type") \
```

Other error messages don't have a period.

Reply:
@targos This appears to be convention for at least `DataCloneError`, though, both on the web and for errors thrown from V8?

Reply:
OK, that's a good reason 👍